examples/understandingUsageStrings.md

```diff
@@ -0,0 +1,86 @@
+|                      `guild` | A [Guild](https://discord.js.org/#/docs/main/master/class/Guild) instance returned from the guild ID.
+|          `user` \| `mention` | A [User](https://discord.js.org/#/docs/main/master/class/User) instance returned from the user ID or mention.
+|                     `member` | A [GuildMember](https://discord.js.org/#/docs/main/master/class/GuildMember) instance returned from the member ID or mention.
+
+> Note: `Literal` is very useful in arguments with multiple options.
+
+___
+
+# Using arguments in your command.
+
+Now, after we understand how to configurate the command, we'll start writting it:
+
+```js
+exports.run = async (client, msg, [...args]) => {
+  // Place Code Here
+};
+```
+
+`[...args]` represents a variable number of arguments give when the command is run. The name of the arguments in the array (and their count) is determined by the `usage` property and its given arguments.
+
+> Note that the commands' arguments are an array. This is a trick called [Destructuring assignment](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
+
+For example, when you have:
+
+```js
+exports.help = {
+  name: "messager",
+  description: "Deletes a message, or edits it.",
+  usage: "<Message:msg> <delete|edit> [newContent:string]",
+  usageDelim: "|",
+  extendedHelp: "",
+};
+```
+
+Then, we have to define all the arguments from *help.usage*, the following code block is an example of how it would look like, and how we would work with them.
+
+```js
+exports.run = async (client, msg, [message, action, newContent]) => {
+  // code
+};
+```
+
+In which `message` is the argument assigned to the message object as provided in `<Message:msg>` argument from usage. Same does `action` for `<delete|edit>` and respectively.
+
+> Keep in mind that we declared `newContent` as an optional argument, if it's not provided, it'll return undefined.
+
+Keep in mind that arguments are delimited by the character or combination of characters written in *help.usageDelim*. In this case, we have assigned the character `|` for it. How do we use this command? Easy:
+
+`komessager 293107496191655936|delete`
+
+The line above will execute the command with the name `messager` (or a command with `messager` as an alias), it'll use [Channel.fetchMessages](https://discord.js.org/#/docs/main/stable/class/TextChannel?scrollTo=fetchMessage) (or [Channel.fetchMessages](https://discord.js.org/#/docs/main/stable/class/TextChannel?scrollTo=fetchMessages) if the bot is a userbot). If the message is not found (you mistyped it or the message is in another channel) it'll warn you that the message hasn't been found. The next argument is a literal, in which must be either `delete` or `edit`. Keep in mind that Komada does *String.toLowerCase()*, if you write `DELETE` or any other variation of [font case](https://techterms.com/definition/font_case), it'll work too.
```

Comment:
This information is outdated. Komada (and Klasa) depends on Discord.js Master (v12).